agents/loco_mc_agent.py

```diff
@@ -110,6 +115,65 @@ def save_object_annotation_to_db(sid, postData):
             logging.debug("in save_object_annotation_to_db, got postData: %r" % (postData))
             saveObjectAnnotationsToDb(self.conn, postData)
 
+        @sio.on("label_propagation")
+        def label_propagation(sid, postData): 
```

Comment:
I am assuming this is only specific to `locobot_agent.py` ? If so, I'd recommend adding this event handler to : `agents/locobot/locobot_agent.py` . 
We should try and add socket events that are common to all droidlet agents to this file.

Reply:
From my understanding, we want label prop to run for any agent that has a detector. Although if `loco_mc_agent.py` is meant for only locobot and the mc agent, then moving this socket event to `locobot_agent.py` would make sense. Can I confirm that `loco_mc_agent.py` is only for these two agents? @anuragprat1k